Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+DisambiguatedPaths.swift

```diff
@@ -10,7 +10,7 @@
 
 import Foundation
 
-private let nonAllowedPathCharacters = CharacterSet.urlPathAllowed.inverted
+private let nonAllowedPathCharacters = CharacterSet.urlPathAllowed.inverted.union(["/"])
```

Comment:
C++ supports overloading the following operators:
```+ - * / % ^ & | ~ ! = < > += -= *= /= %= ^= &= |= << >> >>= <<= == != <= >= <=> && || ++ -- , ->* -> ( ) [ ]```

Reply:
Did you mean to comment on another line? This comment doesn't seem to be about this line.

Reply:
Regardless, assuming that the names for those operators include at least `()`, all of them parse as expected with the current code in this PR.

---

I'm making that assumption because operator definitions in C++ has parameters, for example:
```cpp
void operator ++ () { ... }
MyNumber operator + (const MyNumber& obj) { ... }
MyNumber operator - () { ... }
MyNumber operator - (const MyNumber& obj) { ... }
```

so I would expect the parameters to be included in the symbol name just like they'd be for parameters to functions.

Reply:
Daniel and I talked about this some more offline and agreed to make further changes for C++ operators in a follow up PR. I filed rdar://119616317 to track doing this.